proxy/http2/Http2ConnectionState.cc

```diff
@@ -41,7 +41,7 @@ static const int buffer_size_index[HTTP2_FRAME_TYPE_MAX] = {
   -1,                    // HTTP2_FRAME_TYPE_PRIORITY
   BUFFER_SIZE_INDEX_128, // HTTP2_FRAME_TYPE_RST_STREAM
   BUFFER_SIZE_INDEX_128, // HTTP2_FRAME_TYPE_SETTINGS
-  -1,                    // HTTP2_FRAME_TYPE_PUSH_PROMISE
+  BUFFER_SIZE_INDEX_128, // HTTP2_FRAME_TYPE_PUSH_PROMISE
```

Comment:
`128` is enough size for PUSH_PROMISE frame? It contains Header Block Fragment, so 16K (like HEADERS frame and CONTINUATION frame) is appropriate?